src/app/hooks/use-film-list.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+
+export const useFilmsList = (page:number,isFirstTimeOpen:boolean) => {
+    const [isLoading, setIsLoading] = useState(false);
+    const [films, setFilms]:Array<any> = useState([]);
+    const [error, setError] = useState('');
+    const apiKeyTmdb = <API KEY>;
```

Comment:
Such data as the API keys have to stay in the `.env` file. https://medium.com/how-to-react/using-env-file-in-react-js-b2714235e77e